server/app.js

```diff
@@ -1,7 +1,7 @@
 require('dotenv').config();
 
 const express = require('express');
-const routes = require('./../routes');
+const routes = require('../routes');
```

Comment:
Forgot to mention... `routes` are part of the server, so it would make sense to have this file under the `server` directory.